packages/@react-spectrum/textfield/docs/TextArea.mdx

```diff
@@ -21,6 +21,7 @@ import {Flex} from '@react-spectrum/layout';
 
 ---
 category: Forms
+keywords: [textarea]
```

Comment:
should all of these form elements have any of the keywords 'input'? 'control'? 'field'?

Reply:
sure I'll put `input` in